rest/taskrouter/twiml/example2/example/example.5.x.cs

```diff
@@ -4,21 +4,69 @@
 using System.Net;
 using Twilio.TwiML;
 
-class Example
-{
-    public static HttpListenerResponse SendResponse(HttpListenerContext ctx)
+namespace project {
+    class TwilioExample
     {
-        HttpListenerRequest request = ctx.Request;
-        HttpListenerResponse response = ctx.Response;
+        static void Main(string[] args)
+        {
+            if (!HttpListener.IsSupported)
+            {
+                Console.WriteLine ("HttpListener NOT supported.");
+                return;
+            }
+            string[] prefixes = {"http://*:8080/"};
 
-        response.StatusCode = (int)HttpStatusCode.OK;
-        response.ContentType = "text/xml";
+            // URI prefixes are required,
+            if (prefixes == null || prefixes.Length == 0)
+              throw new ArgumentException("prefixes");
 
-        var twiml = new VoiceResponse();
-        twiml.Enqueue("{\"account_number\":\"12345abcdef\"}",
-            workflowSid: "WW0123456789abcdef0123456789abcdef");
+            // Create a listener.
+            HttpListener listener = new HttpListener();
+            // Add the prefixes.
+            foreach (string s in prefixes)
+            {
+                listener.Prefixes.Add(s);
+            }
+            listener.Start();
+            Console.WriteLine("Listening...");
 
-        response.StatusDescription = twiml.ToString();
-        return response;
+            // Note: The GetContext method blocks while waiting for a request.
+            HttpListenerContext context = listener.GetContext();
+            HttpListenerRequest request = context.Request;
+
+            // Obtain a response object.
+            var result = SendResponse(context);
+            HttpListenerResponse response = result.Item1;
+            string xml = result.Item2;
+
+            byte[] buffer = System.Text.Encoding.UTF8.GetBytes(xml);
+
+            // Get a response stream and write the response to it.
+            response.ContentLength64 = buffer.Length;
+            System.IO.Stream output = response.OutputStream;
+            output.Write(buffer,0,buffer.Length);
+
+            // You must close the output stream.
+            output.Close();
+            listener.Stop();
+
+            Console.WriteLine("Output: \n"+xml);
+        }
+
+        public static (HttpListenerResponse, string) SendResponse(HttpListenerContext ctx)
+        {
+            HttpListenerRequest request = ctx.Request;
+            HttpListenerResponse response = ctx.Response;
+
+            response.StatusCode = (int)HttpStatusCode.OK;
+            response.ContentType = "text/xml";
+
+            var twiml = new VoiceResponse();
+            twiml.Enqueue("{\"account_number\":\"12345abcdef\"}",
+                workflowSid: "WW0123456789abcdef0123456789abcdef");
```

Comment:
This code here is generating this TwiML
```
<Response>
  <Enqueue workflowSid="WW0123456789abcdef0123456789abcdef">{"account_number":"12345abcdef"}</Enqueue>
</Response>
```
and it doesn't match the expected output

Reply:
Thanks @maylonpedroso 
I have edited it now.

If you are happy with this change I will remove all the "web server overkill"

Reply:
I forgot to mention @maylonpedroso 

I think `response.StatusDescription = twiml.ToString();` could be a problem, I did get the error below when testing with that line:

```
Unhandled Exception: System.ArgumentException: Specified value has invalid Control characters.
Parameter name: name
   at System.Net.HttpListenerResponse.set_StatusDescription(String value)
   at project.TwilioExample.SendResponse(HttpListenerContext ctx)
```

This is why I decided to return two values, so I could output the xml properly in the server.

Any thoughts on this?

Reply:
Hi @patillacode,
The exception is because StatusResponse does not support `\n` char in the content. What do you think about doing this:
```
public static void SendResponse(HttpListenerContext ctx)
{
    HttpListenerRequest request = ctx.Request;
    HttpListenerResponse response = ctx.Response;

    response.StatusCode = (int)HttpStatusCode.OK;
    response.ContentType = "text/xml";

    // Create TwiML
    var twiml = new VoiceResponse();
    var enqueue = new Enqueue(workflowSid: "WW0123456789abcdef0123456789abcdef");
    enqueue.Task("{\"account_number\": \"12345abcdef\"}");
    twiml.Append(enqueue);

    // Write the output and close the stream
    byte[] buffer = Encoding.UTF8.GetBytes(twiml.ToString());
    response.ContentLength64 = buffer.Length;
    response.OutputStream.Write(buffer,0, buffer.Length);
    response.OutputStream.Close();
}
```

Reply:
Hello there @maylonpedroso 

Sounds good to me, thanks, I was confused on how much of the _http server_ I could keep in the snippet.

Adding this. Pushing shortly.